composer.json

```diff
@@ -29,6 +29,7 @@
         "league/oauth2-server": "^8.0",
         "league/route": "^4.5",
         "pdsinterop/flysystem-rdf": "^0.1",
+        "pdsinterop/solid-auth": "dev-feature/implicit-grant",
```

Comment:
For future reference, you might want to use [a branch alias](https://getcomposer.org/doc/articles/aliases.md#branch-alias) here.

That way a (future) version can be set here and the branch to work from is set from the `extra` section.